### PR TITLE
MP-5130 Table Feature Row actions

### PR DESCRIPTION
**Pull request by danilmoro3**
Ticket: https://spryker.atlassian.net/browse/MP-5130

### CODE REVIEW

File: docs/marketplace/dev/front-end/table-design/table-feature-extension/table-feature-row-actions.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    TableModule.forRoot(),
+    TableModule.withFeatures({
+      rowActions: () =>
+        import('./table-row-actions-feature.module').then(
+          (m) => m.TableRowActionsFeatureModule,
+        ),    
+    }),
+  ],
+})
+export class RootModule {}
```
Comment (gund) [inline docs/marketplace/dev/front-end/table-design/table-feature-extension/table-feature-row-actions.md:74]: Please move this to a separate section called "Feature Registration" above "Interfaces" section and change the import to the package name not relative import.

File: docs/marketplace/dev/front-end/table-design/table-feature-extension/table-feature-row-actions.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+template: concept-topic-template
+---
+
+This article provides details about the Table Feature Row Actions component in the Components Library.
+
+## Overview
+
+Table Feature Row Actions is a feature of the Table Component that renders dropdown 
+menu that contains actions related specifically to the table row. 
+Also this feature allows triggering actions via row click.
```
Comment (gund) [inline docs/marketplace/dev/front-end/table-design/table-feature-extension/table-feature-row-actions.md:13]: Please also add a link to actions doc.

```suggestion
Table Feature Row Actions is a feature of the Table Component that renders dropdown 
menu that contains actions applicable to the table row and on click triggers an Action which must be registered. 
Also this feature allows triggering actions via row click.
By default all actions will be available on every row but if needed they can be filtered using array of action IDs specified on each row in the data using path configured by `availableActionsPath`.
```

File: docs/marketplace/dev/front-end/table-design/table-feature-extension/table-feature-row-actions.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  columns: [ ... ],
+  rowActions: {
+    enabled: true,
+    actions: [
+      { id: 'edit', title: 'Edit' },
+      { id: 'delete', title: 'Delete' },
+    ],
+    click: 'edit',
+    rowIdPath: 'sku',
+    availableActionsPath: ['edit'],
```
Comment (gund) [inline docs/marketplace/dev/front-end/table-design/table-feature-extension/table-feature-row-actions.md:34]:
```suggestion
    availableActionsPath: ['path.to.actions'],
```

File: docs/marketplace/dev/front-end/table-design/table-feature-extension/table-feature-row-actions.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+Table Feature Row Actions is a feature of the Table Component that renders dropdown 
+menu that contains actions related specifically to the table row. 
+Also this feature allows triggering actions via row click.
+See an example below, how to use the Row Actions feature.
+
+`enabled` - will enable feature via config.  
+`actions` - an array with actions that will be displayed in the dropdown menu.  
+`click` - indicates by `id` which action will be used for the table row clicking.  
+`rowIdPath` - uses for the `rowId` action context.  
+`availableActionsPath` - an array with available actions in the dropdown menu (by `id`).
```
Comment (gund) [inline docs/marketplace/dev/front-end/table-design/table-feature-extension/table-feature-row-actions.md:20]:
```suggestion
`availableActionsPath` - path to an array with available action IDs in the table data row (supports nested objects using dot notation for ex. `prop.nestedProp`)
```

File: docs/marketplace/dev/front-end/table-design/table-feature-extension/table-feature-row-actions.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+```html
+<spy-table [config]="{
+  dataSource: { ... },
+  columns: [ ... ],
+  rowActions: {
+    enabled: true,
+    actions: [
+      { id: 'edit', title: 'Edit' },
+      { id: 'delete', title: 'Delete' },
```
Comment (gund) [inline docs/marketplace/dev/front-end/table-design/table-feature-extension/table-feature-row-actions.md:30]:
```suggestion
      { id: 'edit', title: 'Edit', type: 'edit-action' },
      { id: 'delete', title: 'Delete', type: 'delete-action' },
```

File: docs/marketplace/dev/front-end/table-design/table-feature-extension/table-feature-row-actions.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+## Overview
+
+Table Feature Row Actions is a feature of the Table Component that renders dropdown 
+menu that contains actions related specifically to the table row. 
+Also this feature allows triggering actions via row click.
+See an example below, how to use the Row Actions feature.
+
+`enabled` - will enable feature via config.  
+`actions` - an array with actions that will be displayed in the dropdown menu.
```
Comment (gund) [inline docs/marketplace/dev/front-end/table-design/table-feature-extension/table-feature-row-actions.md:17]:
```suggestion
`actions` - an array with actions that will be displayed in the dropdown menu and their type of registered Action.
```

**Review by gund — CHANGES_REQUESTED** [review]

File: docs/marketplace/dev/front-end/table-design/table-features/table-feature-row-actions.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+  columns: [ ... ],
+  rowActions: {
+    enabled: true,
+    actions: [
+      { id: 'edit', title: 'Edit', type: 'edit-action' },
+      { id: 'delete', title: 'Delete', type: 'delete-action' },
+    ],
+    click: 'edit',
+    rowIdPath: 'sku',
+    availableActionsPath: ['path.to.actions'],
```
Comment (FRITS-Kh) [inline docs/marketplace/dev/front-end/table-design/table-features/table-feature-row-actions.md:40]:
```suggestion
    availableActionsPath: 'path.to.actions',
```

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

**Review by FRITS-Kh — DISMISSED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**